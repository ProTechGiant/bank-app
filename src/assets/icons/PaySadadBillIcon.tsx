import { Path, Svg } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export function PaySadadBillIcon({ color = "#2E2E2E", height = 24, width = 24 }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 20 20" fill={color} xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M16.3203 2.91699L15.0703 1.66699L13.8203 2.91699L12.5703 1.66699L11.3203 2.91699L10.0703 1.66699L8.82031 2.91699L7.57031 1.66699L6.32031 2.91699L5.07031 1.66699V13.3337H2.57031V15.8337C2.57031 17.217 3.68698 18.3337 5.07031 18.3337H15.0703C16.4536 18.3337 17.5703 17.217 17.5703 15.8337V1.66699L16.3203 2.91699ZM12.5703 16.667H5.07031C4.61198 16.667 4.23698 16.292 4.23698 15.8337V15.0003H12.5703V16.667ZM15.9036 15.8337C15.9036 16.292 15.5286 16.667 15.0703 16.667C14.612 16.667 14.237 16.292 14.237 15.8337V13.3337H6.73698V4.16699H15.9036V15.8337Z"
        fill={color}
      />
      <Path d="M12.5703 5.83366H7.57031V7.50033H12.5703V5.83366Z" fill={color} />
      <Path d="M15.0703 5.83366H13.4036V7.50033H15.0703V5.83366Z" fill={color} />
      <Path d="M12.5703 8.33366H7.57031V10.0003H12.5703V8.33366Z" fill={color} />
      <Path d="M15.0703 8.33366H13.4036V10.0003H15.0703V8.33366Z" fill={color} />
    </Svg>
  );
}
