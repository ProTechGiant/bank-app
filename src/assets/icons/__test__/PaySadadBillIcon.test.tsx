import { create } from "react-test-renderer";

import { PaySadadBillIcon } from "../PaySadadBillIcon";

describe("icon/PaySadadBillIcon", () => {
  it("renders a PaySadadBillIcon", () => {
    const result = create(<PaySadadBillIcon />);

    expect(result).toMatchSnapshot();
  });
});
