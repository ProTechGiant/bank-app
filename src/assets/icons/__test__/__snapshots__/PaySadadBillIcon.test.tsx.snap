// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/PaySadadBillIcon renders a PaySadadBillIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="#2E2E2E"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={20}
  vbWidth={20}
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={
      {
        "payload": 4281216558,
        "type": 0,
      }
    }
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M16.3203 2.91699L15.0703 1.66699L13.8203 2.91699L12.5703 1.66699L11.3203 2.91699L10.0703 1.66699L8.82031 2.91699L7.57031 1.66699L6.32031 2.91699L5.07031 1.66699V13.3337H2.57031V15.8337C2.57031 17.217 3.68698 18.3337 5.07031 18.3337H15.0703C16.4536 18.3337 17.5703 17.217 17.5703 15.8337V1.66699L16.3203 2.91699ZM12.5703 16.667H5.07031C4.61198 16.667 4.23698 16.292 4.23698 15.8337V15.0003H12.5703V16.667ZM15.9036 15.8337C15.9036 16.292 15.5286 16.667 15.0703 16.667C14.612 16.667 14.237 16.292 14.237 15.8337V13.3337H6.73698V4.16699H15.9036V15.8337Z"
      fill={
        {
          "payload": 4281216558,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
    <RNSVGPath
      d="M12.5703 5.83366H7.57031V7.50033H12.5703V5.83366Z"
      fill={
        {
          "payload": 4281216558,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
    <RNSVGPath
      d="M15.0703 5.83366H13.4036V7.50033H15.0703V5.83366Z"
      fill={
        {
          "payload": 4281216558,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
    <RNSVGPath
      d="M12.5703 8.33366H7.57031V10.0003H12.5703V8.33366Z"
      fill={
        {
          "payload": 4281216558,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
    <RNSVGPath
      d="M15.0703 8.33366H13.4036V10.0003H15.0703V8.33366Z"
      fill={
        {
          "payload": 4281216558,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
